packages/office-ui-fabric-react/src/components/Dialog/examples/Dialog.TopOffsetFixed.Example.tsx

```diff
@@ -2,105 +2,84 @@ import * as React from 'react';
 import { Dialog, DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
 import { DefaultButton, PrimaryButton } from 'office-ui-fabric-react/lib/Button';
 import { ChoiceGroup } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import { useBoolean } from '@uifabric/react-hooks';
 
-export interface IDialogTopOffsetFixedExampleState {
-  hideDialog: boolean;
-  optionSelected: string;
-}
+const modelProps = {
+  isBlocking: true,
+  topOffsetFixed: true,
+};
+export const DialogTopOffsetFixedExample: React.FunctionComponent = () => {
+  const [hideDialog, { toggle: toggleHideDialog }] = useBoolean(true);
+  const [optionSelected, setOptionSelected] = React.useState('A');
 
-export class DialogTopOffsetFixedExample extends React.Component<{}, IDialogTopOffsetFixedExampleState> {
-  public state: IDialogTopOffsetFixedExampleState = {
-    hideDialog: true,
-    optionSelected: 'A',
+  const onChange = (ev: React.FormEvent<HTMLInputElement>, option: any): void => {
+    setOptionSelected(option.key);
   };
 
-  public render() {
-    const { optionSelected, hideDialog } = this.state;
+  const options = React.useMemo(
```

Comment:
Instead of recalculating the options to set their `checked` state, it's better to define them once (outside the component) and use ChoiceGroup's `selectedKey` prop to set checked state. Not sure why the example was doing it the more complicated way before.